src/style/chronograf.scss

```diff
@@ -143,6 +143,7 @@
 @import 'src/operator/account/AccountGrid.scss';
 @import 'src/writeData/components/fileUploads/FileUploads.scss';
 @import 'src/secrets/components/secrets.scss';
+@import 'src/flows/pipes/Notification/Threshold.scss';
```

Comment:
import this in the notification style instead. This style of css management leads to more dead code